modules/con_enabling_annotation_based_resource_tracking_in_argo_cd.adoc

```diff
@@ -0,0 +1,126 @@
+* If issues arise, check the *Argo CD Application* status and logs in the {OCP} Web Console.
+====
+
+.Pre-requisites
+
+* You have logged in to the {OCP} cluster as an administrator.
+* You have installed the {gitops-title} Operator on your {OCP} cluster.
+
+[id="configuring-annotation-based-tracking-in-multiple-argo-cd-instances_{context}"]
+== Configuring annotation-based tracking in multiple Argo CD instances
+
+You can configure annotation-based tracking in multiple Argo CD instances by completing the following steps.
+
+. Log in to the {OCP} web console. 
+
+. In the *Administrator* perspective of the web console, click *Operators* -> *Installed Operators*.
+
+. Create or select the project where you want to install the user-defined Argo CD instance from the *Project* list.
+
+. Select *{gitops-title}* from the installed Operators list and click the *Argo CD* tab.
+
+. Click *Create ArgoCD* to create two Argo CD instances using the following YAML snippet.
++
+.Example Argo CD instance with an annotation label
+[source,yaml]
+----
+apiVersion: argoproj.io/v1beta1
+kind: ArgoCD
+metadata:
+  name: argocd-instance-demo-1 # <1>
+  namespace: argocd-test-demo-1 # <2>
+  name: argocd-instance-demo-2 # <3>
+  namespace: argocd-test-demo-2 # <4>
+spec:
+  installationID: "instance-demo-1" # <5>
+  resourceTrackingMethod: "annotation+label"
+  installationID: "instance-demo-2" # <6>
+  resourceTrackingMethod: "annotation+label"
+----
++
+<1> Specifies the name of the first Argo CD instance.
+<2> Specifies the namespace used for the first Argo CD instance.
+<3> Specifies the name of the second Argo CD instance.
+<4> Specifies the namespace used for the second Argo CD instance.
+<5> Specifies the name of the `installationID` for the first Argo CD instance.
+<6> Specifies the name of the `installationID` for the second Argo CD instance.
++
+. Configure and label target namespaces to associate namespaces with their Argo CD instances.
+.. Navigate to *Administration* -> *Namespaces*.
+.. Create namespaces for application deployments, `app-ns-1` and `app-ns-2`.
+.. To associate each namespace with their respective Argo CD instances, run the following command:
++
+[source,terminal]
+----
+oc label namespace app-ns-1 argocd.argoproj.io/managed-by=argocd-test-demo-1
+oc label namespace app-ns-2 argocd.argoproj.io/managed-by=argocd-test-demo-2
+----
++
+. Create two applications in Argo CD.
+.. In the OpenShift Web Console, go to *Operators* → *Installed Operators* → *OpenShift GitOps Operator*.
+.. Select *Argo CD* and navigate to the *Applications* tab.
+.. Click *Create Application*.
+.. Enter the following YAML snippet to create two applications in Argo CD.
++
+.Example application creation using Argo CD
+[source,yaml]
+----
+apiVersion: argoproj.io/v1alpha1
+kind: Application
+metadata:
+  name: sprint-petclinic # <1>
+  namespace: argocd-test-1 # <2>
+  name: sprint-petclinic # <3>
```

Comment:
Can't club 2 applications like this. This will cause error. Instead have 2 seperate `Application` objects.

Reply:
Fixed.